src/core/router/routes.tsx

```diff
@@ -0,0 +1,85 @@
+import { lazily } from 'react-lazily';
+import {
+  Alarm24,
+  Info24,
+  Map24,
+  Prefs24,
+  User24,
+  Reports16,
+} from '@konturio/default-icons';
+import { i18n } from '~core/localization';
+import { AppFeature } from '~core/auth/types';
+import { HeaderTitle } from '~components/HeaderTitle/HeaderTitle';
+import { UserStateToComponents } from '~core/auth';
+import appConfig from '~core/app_config';
+import history from './history';
+import type { AppRouterConfig } from './types';
+
+const { MapPage } = lazily(() => import('~views/Map/Map'));
+const { ReportsPage } = lazily(() => import('~views/Reports/Reports'));
+const { ReportPage } = lazily(() => import('~views/Report/Report'));
+const { ProfilePage } = lazily(() => import('~views/Profile/Profile'));
+const { AboutPage } = lazily(() => import('~views/About/About'));
+const { BivariateManagerPage } = lazily(
+  () => import('~views/BivariateManager/BivariateManager'),
+);
+
+export const getAbsoluteRoute = (slug: string) => appConfig.baseUrl + slug;
+export const toHomePage = () => history.push(routerConfig.defaultRoute);
+export const routerConfig: AppRouterConfig = {
+  defaultRoute: 'map',
+  routes: [
+    {
+      slug: 'about',
+      title: i18n.t('modes.about'),
+      icon: <Info24 />,
+      view: <AboutPage toHomePage={toHomePage} />,
+      showForNewUsers: true,
+    },
+    {
+      slug: 'profile',
+      title: (
+        <UserStateToComponents
+          authorized={i18n.t('modes.profile')}
+          other={i18n.t('login.login_button')}
+        />
+      ),
+      icon: <User24 />,
+      view: <ProfilePage />,
+      requiredPermission: AppFeature.APP_LOGIN,
```

Comment:
required feature name? It's wierd that _permission_ is a string

Reply:
why its wired?

Reply:
required permission renamed to requiredFeature